docs/docsite/rst/dev_guide/developing_collections_migrating.rst

```diff
@@ -13,18 +13,19 @@ When you move content from one collection to another, for example to extract a s
 Migrating content
 =================
 
-Before you start migrating content from one collection to another, look at `Ansible Collection Requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html>`_.
-
-To migrate content from one collection to another, if the collections are parts of `Ansible distribution <https://github.com/ansible-community/ansible-build-data/blob/main/2.10/ansible.in>`_:
+If the collection you are going to migrate content from is a part of the `Ansible community package <https://github.com/ansible-community/ansible-build-data/blob/main/>`_, make sure the target collection satisfies the `Ansible collection requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html>`_. After that, to migrate the content:
 
 #. Copy content from the source (old) collection to the target (new) collection.
+#. Change ``M()``, examples, ``seealso``, ``extended_documentation_fragments`` to use actual FQCNs in moved content and in other collections that have references to the content.
+#. Move all related issues, pull requests, and wiki pages.
+#. Look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using the ``grep`` command-line utility) to check if there are examples using the moved modules and plugins to update their FQCNs.
```

Comment:
```suggestion
#. Look through the ``docs/docsite`` directory of the `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using the ``grep`` command-line utility) to check if there are examples using the moved modules and plugins so that you can update those FQCNs.
```